package.yaml

```diff
@@ -27,14 +27,17 @@ default-extensions:
   - DefaultSignatures
   - DeriveDataTypeable
   - DeriveGeneric
+  - DerivingStrategies
   - FlexibleContexts
   - FlexibleInstances
   - FunctionalDependencies
   - GeneralizedNewtypeDeriving
+  - ImportQualifiedPost
```

Comment:
Now stylish-haskell automatically rewrites `import A qualified as B` to `import qualified A as B`.

You can verify this behaviour by manually running this:

```
stylish-haskell --inplace src/Xrefcheck/Scanners/Markdown.hs
```

If we wanna use `ImportQualifiedPost`, we need to configure stylish-haskell to keep the "ImportQualifiedPost" format.

Reply:
![image](https://user-images.githubusercontent.com/1784505/140088224-2fd0f92a-84e3-4d4f-b3a9-9c2b5cb0458c.png)

Google finds 0 results though. I'll try to do it, but can't guarantee it is possible.

Reply:
Disregard that, it is possible https://github.com/akrmn/stylish-haskell/commit/c5c7b50445e930389a22e303ee0d84679af8a93e#diff-0924d0340532d02b8c3bacd84f133f5c6da1bf2494e872247f3abfc9e1c8d4b1